variables.tf

```diff
@@ -142,4 +175,6 @@ variable "s3_version" {
 }
 
 variable "vcenter_iso_name" {
+  description = "The name of the vCenter ISO in your Object Store"
```

Comment:
Nice description addition. 

Terraform modules should include descriptions for all of the variables and outputs.

Reply:
Yeah. Let me double check I have all of these.

Reply:
I just added a bunch of descriptions.